roles/docker_storage_to_overlay2/vars/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+storage_setup_path: "/etc/sysconfig/docker-storage-setup"
+
+# List of services to stop and restart.  Any available services
+# in optional_service_names get appended to this list beforehand.
+service_names: []
+
+optional_service_names:
+- "docker.service"            # } One or the other
```

Comment:
No change needed but openshift_facts does set ``openshift.docker.service_name`` which will expand to either ``docker.service`` or ``container-enginer.service``.

Reply:
:+1: Nice, that's handy.  Also see my question above about `docker info`.